abi-bindings/go/Utilities/ValidatorSetSig/packing_test.go

```diff
@@ -0,0 +1,31 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package validatorsetsig
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPackUnpackValidatorSetSigMessage(t *testing.T) {
+	msg := ValidatorSetSigMessage{
```

Comment:
Not for this PR, but in all our `packing_test.go`'s it would be great if we could perform validation on the struct in the test itself. For instance, if we add a new field to the Solidity struct under test, this Go test would still pass. https://github.com/GaijinEntertainment/go-exhaustruct seems like a good candidate, and it's included in golang-ci, which we already use. I'm not sure if you can target specific linters at specific files (but not others) with golang-ci, but a simple alternative would be running exhaustruct directly.

Reply:
👍 cut an issue for it #411